spec/grape/api_spec.rb

```diff
@@ -1723,6 +1723,32 @@ class CustomError < Grape::Exceptions::Base; end
       expect(last_response.status).to eql 500
       expect(last_response.body).to eq('Formatter Error')
     end
+
+    context 'validates response processed by exception handler' do
+      it 'calls default_rescue_handler when response is invalid' do
+        subject.rescue_from :all do
+          error!('Internal Server Error')
+          nil # invalid response caused by return nil
```

Comment:
@dblock I’m not sure if I get what you mean.

> Are we breaking any existing specs if we start ignoring the response from rescue_from altogether? If the answer is no, then we should just do that and document it.

Do you mean if we start ignoring what returned from `rescue_from`? If so, there would be only one spec broken, which I added it here.